### ISSUE TITLE
There is no sound in the bluetooth headphones of galaxy buds when watching streams

### ISSUE THREAD

**Issue by ilyavaiser** [TelegramMessenger/Telegram-iOS#748]
### Checklist

- [x] I am reporting an issue in existing functionality that does not work as intended
- [x] I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description

When using Galaxy Buds bluetooth headphones of different models, the sound does not work in telegram streams. 

### Expected Behavior

I have to hear the sound of the stream.

### Actual Behavior

Sound appears for a split second when using Galaxy Buds headphones and connected to the stream, and then sound stops working, When using Apple Airpods headphones, everything works fine.

### Steps to Reproduce

1. Connect Galaxy Buds headphones to the iphone
2. Enter into the telegram public
3. Open the stream

### Environment

**Device:** `iPhone 11 Pro Max`

**iOS version**: `15.14.1`

**App version:** `8.6`

**Comment by AgranatMarkit** [comment]
Please fix that's bug, it's very annoing.
I couldn't listen streams on my phone.

**Comment by Mikry13** [comment]
I have the same problem on latest iOS :(

**Comment by asketsus** [comment]
Do we have some update on this issue?

**Comment by ilyavaiser** [comment]
No, Pasha Durov trains his muscles and draws new emoticons. They don't seem to solve any real problems at all. For the last six months, I can’t even make a normal telegram call, unlike Whatsapp, I have to constantly switch to it

**Comment by ericguti** [comment]
It happens also with the Huawei Freebuds 4i
